test/rules/unused_callbacks/examples/bad.exs

```diff
@@ -2,7 +2,7 @@
 defmodule MeandroTest.Examples.UnusedCallbacks.Bad do
   @callback used(atom()) :: :res_used
   @callback used_too(atom()) :: :res_used_too
-  @callback unused() :: :res_unused
+  @callback unused :: :res_unused
```

Comment:
I don't like this... is credo suggesting that?

Reply:
No. It's a test module. We need to test that this is parsed correctly, since it's valid code.